vmngclient/endpoints/tenant_migration.py

```diff
@@ -48,9 +48,11 @@ def export_tenant_data(self, payload: Tenant) -> ExportInfo:
     def get_migration_token(self, params: MigrationTokenQueryParams) -> str:
         ...
 
-    def import_tenant_data(self, data: BinaryIO) -> ImportInfo:
+    def import_tenant_data(self, migrationKey, data: BinaryIO) -> ImportInfo:
```

Comment:
Introduced parameter is not used in method body (It is not reaching vManage in any way)